docs/StardustDocs/topics/toHTML.md

```diff
@@ -2,13 +2,222 @@
+
+```kotlin
+val htmlData = dataFrameOf(
+    "kotlinLogo" to columnOf(
+        IMG("https://kotlin.github.io/dataframe/images/kotlin-logo.svg"),
+    ),
+).toStandaloneHtml()
+```
+
+<!---END-->
+
+### Embed pages
+
+Values of an `org.jetbrains.kotlinx.dataframe.datatypes.IFRAME` class are rendered as `<iframe>` tag
+
+<!---FUN displayIFrame-->
+
+```kotlin
+val htmlData = dataFrameOf(
+    "documentationPages" to columnOf(
+        IFRAME(
+            src = "https://kotlin.github.io/dataframe/tohtml.html",
+            width = 850,
+            height = 500,
+        ),
+    ),
+).toStandaloneHtml()
+```
+
+<!---END-->
+
+### Render clickable links
+
+Values of `java.net.URL` are rendered as `<a>` tag
+
+<!---FUN displayURL-->
+
+```kotlin
+val htmlData = dataFrameOf(
+    "documentationPages" to columnOf(
+        URI("https://kotlin.github.io/dataframe/format.html").toURL(),
+        URI("https://kotlin.github.io/dataframe/tohtml.html").toURL(),
+        URI("https://kotlin.github.io/dataframe/jupyterrendering.html").toURL(),
+    ),
+)
+    .toStandaloneHtml()
+```
+
+<!---END-->
+
+### Render any HTML inside a cell
+
+Wrap cell values in a custom HTML using `RenderedContent.media`
+
+<!---FUN displayMediaContent-->
+<tabs>
+<tab title="Properties">
+
+```kotlin
+val htmlData = dataFrameOf(
+    "documentationPages" to columnOf(
+        "https://kotlin.github.io/dataframe/format.html",
+        "https://kotlin.github.io/dataframe/tohtml.html",
+        "https://kotlin.github.io/dataframe/jupyterrendering.html",
+    ),
+)
+    .convert { documentationPages }.with {
+        val uri = URI(it)
+        RenderedContent.media("""<a href='$uri'>${uri.path}</a>""")
+    }
+    .toStandaloneHtml()
+```
+
+</tab>
+<tab title="Strings">
+
+```kotlin
+val htmlData = dataFrameOf(
+    "documentationPages" to columnOf(
+        "https://kotlin.github.io/dataframe/format.html",
+        "https://kotlin.github.io/dataframe/tohtml.html",
+        "https://kotlin.github.io/dataframe/jupyterrendering.html",
+    ),
+)
+    .convert { "documentationPages"<String>() }.with {
+        val uri = URI(it)
+        RenderedContent.media("""<a href='$uri'>${uri.path}</a>""")
+    }
+    .toStandaloneHtml()
+```
+
+</tab></tabs>
+<!---END-->
+
+### Sample data 
+This dataframe is used in the following examples
+
+<!---FUN df-->
+
+```kotlin
+val df = dataFrameOf(
+    "name" to columnOf(
+        "firstName" to columnOf("Alice", "Bob", "Charlie", "Charlie", "Bob", "Alice", "Charlie"),
+        "lastName" to columnOf("Cooper", "Dylan", "Daniels", "Chaplin", "Marley", "Wolf", "Byrd"),
+    ),
+    "age" to columnOf(15, 45, 20, 40, 30, 20, 30),
+    "city" to columnOf("London", "Dubai", "Moscow", "Milan", "Tokyo", null, "Moscow"),
+    "weight" to columnOf(54, 87, null, null, 68, 55, 90),
+    "isHappy" to columnOf(true, true, false, true, true, false, true),
+)
+```
+
+<!---END-->
+
+### Reusable rendering logic
+
+Generic approach to custom cell rendering
```

Comment:
I miss a tiny explanation of why you'd want to do this as a user. When is this useful?